classes/class-admin-notices.php

```diff
@@ -0,0 +1,1030 @@
+			return;
+		}
+		if ( 'success' === $_GET['ewww_pngout'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+			?>
+			<div id='ewww-image-optimizer-pngout-success' class='notice notice-success fade'>
+				<p>
+					<?php \esc_html_e( 'Pngout was successfully installed.', 'ewww-image-optimizer' ); ?>
+				</p>
+			</div>
+			<?php
+		}
+		if ( 'failed' === $_GET['ewww_pngout'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+			?>
+			<div id='ewww-image-optimizer-pngout-failure' class='notice notice-error'>
+				<p>
+					<?php
+					\printf(
+						/* translators: 1: An error message 2: The folder where pngout should be installed */
+						\esc_html__( 'Pngout was not installed: %1$s. Make sure this folder is writable: %2$s', 'ewww-image-optimizer' ),
+						( ! empty( $_GET['ewww_error'] ) ? \esc_html( \sanitize_text_field( \wp_unslash( $_GET['ewww_error'] ) ) ) : \esc_html__( 'unknown error', 'ewww-image-optimizer' ) ), // phpcs:ignore WordPress.Security.NonceVerification
+						\esc_html( EWWW_IMAGE_OPTIMIZER_TOOL_PATH )
+					);
+					?>
+				</p>
+			</div>
+			<?php
+		}
+	}
+
+	/**
+	 * Display a success or failure message after SVGCLEANER installation.
+	 */
+	public function svgcleaner_installed_notice() {
+		if ( empty( $_GET['ewww_svgcleaner'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+			return;
+		}
+		if ( 'success' === $_GET['ewww_svgcleaner'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+			?>
+			<div id='ewww-image-optimizer-pngout-success' class='notice notice-success fade'>
+				<p>
+					<?php \esc_html_e( 'Svgcleaner was successfully installed.', 'ewww-image-optimizer' ); ?>
+				</p>
+			</div>
+			<?php
+		}
+		if ( 'failed' === $_GET['ewww_svgcleaner'] ) { // phpcs:ignore WordPress.Security.NonceVerification
+			?>
+			<div id='ewww-image-optimizer-pngout-failure' class='notice notice-error'>
+				<p>
+					<?php
+					\printf(
+						/* translators: 1: An error message 2: The folder where svgcleaner should be installed */
+						\esc_html__( 'Svgcleaner was not installed: %1$s. Make sure this folder is writable: %2$s', 'ewww-image-optimizer' ),
+						( ! empty( $_GET['ewww_error'] ) ? \esc_html( \sanitize_text_field( \wp_unslash( $_GET['ewww_error'] ) ) ) : \esc_html__( 'unknown error', 'ewww-image-optimizer' ) ), // phpcs:ignore WordPress.Security.NonceVerification
+						\esc_html( EWWW_IMAGE_OPTIMIZER_TOOL_PATH )
+					);
+					?>
+				</p>
+			</div>
+			<?php
+		}
+	}
+
+	/**
+	 * Remind network admin to activate Easy IO on the new site.
+	 */
+	public function easyio_site_initialized_notice() {
+		if (
+			! \str_contains( \add_query_arg( '', '' ), 'site-new.php' ) ||
+			empty( $_GET['update'] ) || // phpcs:ignore WordPress.Security.NonceVerification
+			'added' !== $_GET['update'] || // phpcs:ignore WordPress.Security.NonceVerification
+			empty( $_GET['id'] ) // phpcs:ignore WordPress.Security.NonceVerification
+		) {
+			return;
+		}
```

Comment:
It appears the 'update' param is no longer present when adding a new site, so those 2 conditionals need to go.
However, we should confirm two things:
The EASYIO_NEW_SITE_AUTOREG check should also include a check for an API key:
`ewww_image_optimizer_get_option( 'ewww_image_optimizer_cloud_key' )`

And the else should become an elseif, checking to see if the primary/first site is using Easy IO:
`get_option( 'ewww_image_optimizer_exactdn' );`

I think that should do it, so we'll try that out and see!